pkg/agent/agent.go

```diff
@@ -226,50 +226,94 @@ func (i *Initializer) initInterfaceStore() error {
 		return err
 	}
 
+	parseGatewayIFFunc := func(port *ovsconfig.OVSPortData, ovsPort *interfacestore.OVSPortConfig) *interfacestore.InterfaceConfig {
```

Comment:
Could we write shorthand of single word in camel case to be differentiated from acronyms?
For example, we use svc instead of SVC for service, use ch instead of CH for channel, use cfg instead CFG for configration.